src/App.css

```diff
@@ -0,0 +1,120 @@
+* {
```

Comment:
- [ ] Good job so far your app is impressive :clap:  But you need to work on your app design. From the project requirement, you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including Typography: font face, size, and weight and Layout: composition and space between elements. You can check the screenshots below :arrow_down: to see the differences. As a developer you need to pay attention to details :+1: 

#### Expected design
![metrics_app(1)](https://user-images.githubusercontent.com/6615394/211091065-a58853ea-2005-4742-8b15-a532f6322253.jpg)
 
#### Your design
![metricss](https://github.com/forLoop94/Population-Metrics-App/assets/6615394/6a0787e1-bb63-4f0d-ba1c-e193146d2cbf)